From e9645ecc2bd48a9b80ef4ff8038f7ad9c6e1f1a4 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Mon, 28 Sep 2015 13:53:47 +0800
Subject: [PATCH 986/987] iommu: remove the limitation which iommu is
 unavailable for PPC64

The iommu driver is also used for e6500/e5500 core which is PPC64,
so remove the limitation which iommu is only used for PPC32.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/iommu/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/iommu/Kconfig b/drivers/iommu/Kconfig
index 1ae4e54..33b6345 100644
--- a/drivers/iommu/Kconfig
+++ b/drivers/iommu/Kconfig
@@ -50,7 +50,6 @@ config OF_IOMMU
 
 config FSL_PAMU
 	bool "Freescale IOMMU support"
-	depends on PPC32
 	depends on PPC_E500MC || COMPILE_TEST
 	select IOMMU_API
 	select GENERIC_ALLOCATOR
-- 
1.9.1

